### DISCUSSION TITLE
git hub

### DISCUSSION THREAD

**Discussion by Nandani-chauhan** [Jay-study-nildana/FrontEndForStudents#36] [Coding Questions]
sir I have an issue .  the commit command isn't working in my laptop. 
I typed git commit and then  press enter so showing me:-( fatal : unable to auto detect e-mail address) this line .
how I fix it ? plz tell me

**Comment by Jay-study-nildana** [comment]
@Nandani-chauhan 

Ah, you are actively asking questions. very good. very good. 

Option1: 

Now, in the other question you asked, I have given link to youtube videos. 

https://github.com/Jay-study-nildana/FrontEndForStudents/discussions/35

So, one of the video has the solution to this question. I sincerely request to try watching that video, especially the video about "virtual machine" which has this step explained and completed in detail. 

Option2:

If you read the error message very carefully, the solution is already given in the error message. There will be two lines that you have to copy paste. 

First line, you have to very carefully enter your email address. 
Second line, you have to very carefully enter your name. 

---

And, if you are super serious about learning to code, start using discord and get connected with me. my discord details are available in my post signature. 

---

Also, asking questions on github, there is a good technique. look at this link to learn how to ask questions better and become a better developer. 

https://github.com/Jay-study-nildana/FrontEndForStudents/discussions/25

---

Happy coding and keep asking questions and keep learning. 

Jay (Vijayasimha BR)
Coding Tutor. Career and Life Coach
----
Discord: Jay_HisManyHobbies#6392
